jujugui/static/gui/src/app/store/env/controller-api.js

```diff
@@ -463,7 +463,6 @@ YUI.add('juju-controller-api', function(Y) {
       Return information about Juju models, such as their names, series, and
       provider types, by performing a ModelManager.ModelInfo Juju API request.
 
-      @method modelInfo
```

Comment:
It's not required anymore, and that's a good thing, as the name of the method/function is obvious without the annotation, and removing it reduces noise and means one less thing that can go out of sync (we have many examples of methods changing name but not the docstring). So I decided to remove them when I encounter them.